File: web/src/types/LeaseSummary.ts
export interface LeaseSummary {
    occupancy: string;
    leasingStatus: null;
    marketedRent: null;
    paymentStatus: null | string;
    leaseStartDate: string;
    leaseEndDate: string;
    monthlyRent: number;
    securityDepositAmount: number | null;
    hasPet: null;
    petFeeAmount: null;
    isPetsDeposit: boolean;
    petsDepositAmount: number | null;
    isLeaseConcessions: boolean | null;
    isRentersInsuranceRequired: boolean;
    isSection8: boolean;
    isTenantBackgroundChecked: boolean;
    isTenantIncomeAbove3x: boolean;
    isTenantMayTerminateEarly: null;
    isTenantPurchaseOption: null;
}

File: backend/RoofstockSolution/RoofstockSolution/LeaseSummary.cs
using System;

namespace RoofstockSolution
{
    public partial class LeaseSummary
    {
        public string Occupancy { get; set; }
        public string LeasingStatus { get; set; }
        public decimal? MarketedRent { get; set; }
        public string PaymentStatus { get; set; }
        public DateTime? LeaseStartDate { get; set; }
        public DateTime? LeaseEndDate { get; set; }
        public decimal? MonthlyRent { get; set; }
        public decimal? SecurityDepositAmount { get; set; }
        public bool? HasPet { get; set; }
        public decimal? PetFeeAmount { get; set; }
        public bool? IsPetsDeposit { get; set; }
        public decimal? PetsDepositAmount { get; set; }
        public bool? IsLeaseConcessions { get; set; }
        public bool? IsRentersInsuranceRequired { get; set; }
        public bool? IsSection8 { get; set; }
        public bool? IsTenantBackgroundChecked { get; set; }
        public bool? IsTenantIncomeAbove3x { get; set; }
        public bool? IsTenantMayTerminateEarly { get; set; }
        public bool? IsTenantPurchaseOption { get; set; }
    }
}